src/main/java/io/pivotal/cla/data/repository/IndividualSignatureRepository.java

```diff
@@ -35,7 +35,7 @@ default IndividualSignature findSignaturesFor(User user,  String claName) {
 		return results.isEmpty() ? null : results.get(0);
 	}
 
-	@Query("select s from IndividualSignature s where (s.cla.name = :#{#claName} or s.cla.name in (select distinct c.supersedingCla.name from ContributorLicenseAgreement c where c.name = :#{#claName})) and (s.githubLogin = :#{#u.githubLogin} or s.email in (:#{#u.emails}))")
+	@Query("select s from IndividualSignature s where (s.cla.name = :#{#claName} or s.cla.name in (select distinct c.supersedingCla.name from ContributorLicenseAgreement c where c.name = :#{#claName})) and (s.githubLogin = :#{#u.githubLogin} or s.email in (:#{#u.emails.empty ? '' : #u.emails}))")
```

Comment:
How is the random string any less likely than en empty String? Values must be a valid email from the GitHub API. So it seems unlikely we would get anything that doesn't contain a `@` character.